src/fairplay.js

```diff
@@ -128,7 +128,7 @@ export const defaultGetCertificate = (fairplayOptions) => {
   };
 };
 
-export const defaultGetContentId = (emeOptions, initData) => {
+export const defaultGetContentId = (initData) => {
```

Comment:
Technically `getContentId` is an exposed API (from the README), and `emeOptions` should be passed in case someone passes a custom `getContentId`

Reply:
yeah, we can't change this method signature or it'll be a breaking change.

Reply:
fixed